build.sbt

```
@@ -355,13 +371,15 @@ lazy val examples = project
   .settings(allSettings)
   .settings(noPublish)
   .settings(resolvers += "TM" at "https://maven.twttr.com")
-  .settings(coverageExcludedPackages :=
-    """
-      |io\.finch\.div\..*;
-      |io\.finch\.todo\..*;
-      |io\.finch\.wrk\..*;
-      |io\.finch\.iteratee\..*;
-    """.stripMargin)
+  .settings(
+    coverageExcludedPackages :=
+      """
+        |io\.finch\.div\..*;
+        |io\.finch\.todo\..*;
+        |io\.finch\.wrk\..*;
+        |io\.finch\.iteratee\..*;
+    """.stripMargin
```

Weird multi-line string formatting
```suggestion
      """
        |io\.finch\.div\..*;
        |io\.finch\.todo\..*;
        |io\.finch\.wrk\..*;
        |io\.finch\.iteratee\..*;
      """.stripMargin
```